IdentityCore/src/oauth2/aad_base/MSIDAADWebviewFactory.m

```diff
@@ -123,6 +123,17 @@ - (MSIDWebviewResponse *)oAuthResponseWithURL:(NSURL *)url
 #if AD_BROKER
     MSIDCBAWebAADAuthResponse *cbaResponse = [[MSIDCBAWebAADAuthResponse alloc] initWithURL:url context:context error:nil];
     if (cbaResponse) return cbaResponse;
+    
+    if ([url.absoluteString containsString:@"sso_nonce="])
+    {
+        NSString *ssoNonce = [[url msidQueryParameters] valueForKey:@"sso_nonce"];
```

Comment:
I see you hardcode sso_nonce string in multiple places. Can you move it to constants?

Reply:
Yes, moved it to constants and updated related places with it.